src/Modules/ProjectModule/components/ProjectList/ProjectList.tsx

```diff
@@ -1,6 +1,135 @@
+                  onChange={getTitleValue}
+                />
+              </div>
+              <div className="table-responsive-sm table-responsive-md">
+                <table className="table table-striped text-center w-100">
+                  <thead>
+                    <tr>
+                      <th scope="col">#</th>
+                      <th scope="col">Title</th>
+                      <th scope="col">Date Creation</th>
+                      <th scope="col">Modification Date</th>
+                      <th scope="col">Description</th>
+                      <th scope="col">Actions</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {ProjectsList?.map((project: any, index) => (
+                      <tr key={project.id} >
+                        <td>
+                          {project.id}
+                        </td>
+                        <td>{project.title}</td>
+                        <td>
+                          {new Date(project.creationDate).toLocaleDateString(
+                            "en-US"
+                          )}
+                        </td>
+                        <td>
+                          {new Date(
+                            project.modificationDate
+                          ).toLocaleDateString("en-US")}
+                        </td>
+                        <td>{project.description}</td>
+                        <td className="position-relative">
+                          <i
+                            onClick={() => handleShowing(index)}
+                            className="fa-solid fa-ellipsis"
+                          ></i>
+                          <div className="position-absolute icons ">
+                            {showIconIndex === index && (
+                              <div className="icon-container d-flex flex-column bg-white p-3 rounded-3 shadow-sm">
+                                <i className="fa-regular fa-eye text-success my-1"></i>
+                                <i className="fa-solid fa-trash text-danger my-1"></i>
+                                <i className="fa-solid fa-pen-to-square text-warning my-1"></i>
+                              </div>
+                            )}
+                          </div>
+                        </td>
+                      </tr>
+                    ))}
+                  </tbody>
+                </table>
+
+                <nav aria-label="Page navigation example">
+                  <ul className="pagination justify-content-end">
+                    <li className="page-item disabled">
+                      <a className="page-link">Previous</a>
+                    </li>
+
+                    {arrayOfPages.map((pageNo) => 
+                      <li key={pageNo} className="page-item" onClick={() => getProjectsList(titleValue , 10 ,pageNo)}>
+                        <a className="page-link text-secondary">
```

Comment:
This can be changed to a button and can be given the onClick instead of giving the onClick to non interactive element such as `li`